ci-operator/step-registry/baremetalds/e2e/upgrade/upgrade/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
+- andfasano
+- eisraeli
```

Comment:
Please add also the other baremetalds owners to the list

Reply:
+1